test/services/helpers/registration.test.js

```diff
@@ -88,7 +88,7 @@ describe('registration service', () => {
 				});
 			}).catch((err) => {
 				chai.expect(err).to.not.equal(undefined);
-				chai.expect(err.message).to.equal('Ungültige Pin, bitte überprüfe die Eingabe.');
+				chai.expect(err.message).to.equal('Der eingegebene Code konnte leider nicht verfiziert werden. Versuch es doch noch einmal.');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 142. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5203912)